File: app/src/main/java/tech/diarmaid/koohiiaite/activity/PrimitiveListActivity.java
package tech.diarmaid.koohiiaite.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.GridView;

import tech.diarmaid.koohiiaite.R;
import tech.diarmaid.koohiiaite.adapter.PrimitiveGridAdapter;

/**
 * View and rename heisig primitives in a grid view
 */
public class PrimitiveListActivity extends AppCompatActivity {
    GridView gridView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_primitives_list);

        gridView = findViewById(R.id.primitive_grid);
        gridView.setAdapter(new PrimitiveGridAdapter(this));
    }
}

File: app/src/main/java/tech/diarmaid/koohiiaite/activity/PrimitiveListActivity.kt
package tech.diarmaid.koohiiaite.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import tech.diarmaid.koohiiaite.adapter.PrimitiveGridAdapter
import tech.diarmaid.koohiiaite.databinding.ActivityPrimitivesListBinding

/**
 * View and rename heisig primitives in a grid view
 */
class PrimitiveListActivity : AppCompatActivity() {

    private lateinit var binding: ActivityPrimitivesListBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPrimitivesListBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        binding.primitiveGrid.adapter = PrimitiveGridAdapter(this)
    }
}
